examples/async/async.metatags

```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates the usage of async functionality in Ballerina, a way to execute operations asynchronously by returning a `future`.
+keywords: ballerina, ballerina-lang, ballerina by example, bbe, async, future, wait
```

Comment:
Suggested edits: keywords: ballerina, ballerina by example, bbe, async, asynchronous, future, wait

(Since we are already optimized for Ballerinalang, we don't need to add that here)

Reply:
Remove ballerina-lang from other BBEs as well